manifests/user-roles.jsonnet

```diff
@@ -9,26 +9,11 @@ local kubecfg = import "kubecfg.libsonnet";
     // Gives read-only access to Applications within a Namespace in Kubeapps.
     // Usage:
     //   Apply kubeapps-applications-read clusterrole to user/serviceaccount in the desired namespace
-    //   AND apply kubeapps-tiller-state-read role to user/serviceaccount in the kubeapps namespace.
```

Comment:
They are not that broad since it gives access just to a specific namespace (which is logical in a multi-tenancy environment). I think it is nice to define them as far as users know what it does (and it's explained in the documentation).

Reply:
@andresmgot why do we need a role though? We could document users to do `kubectl create rolebinding --clusterrole=cluster-admin --serviceaccount default:kubeapps-operator --namespace default`, right?